sdk/identity/azure-identity/README.md

```diff
@@ -0,0 +1,186 @@
+import com.azure.security.keyvault.keys.KeyClient;
+
+// authenticate with client secret,
+ClientSecretCredential clientSecretCredential = new ClientSecretCredential()
+	    .clientId("<YOUR_CLIENT_ID>")
+	    .clientSecret("<YOUR_CLIENT_SECRET>")
+	    .tenantId("<YOUR_TENANT_ID>");
+
+KeyClient client = KeyClient.builder()
+    .endpoint("https://{YOUR_VAULT_NAME}.vault.azure.net")
+    .credential(clientSecretCredential)
+    .build();
+```
+
+## Chaining credentials:
+The `ChainedTokenCredential` class provides the ability to link together multiple credential instances to be tried sequentially when authenticating. The following example demonstrates creating a credential which will attempt to authenticate using managed identity, and fall back to certificate authentication if a managed identity is unavailable in the current environment. This example authenticates an `EventHubClient` from the [azure-eventhubs][eventhubs_client_library] client library using the `ChainedTokenCredential`.
+
+```java
+import com.azure.identity.credential.ClientSecretCredential;
+import com.azure.security.keyvault.secrets.SecretClient;
+
+ClientSecretcredential firstServicePrincipal = new ClientSecretCredential()
+	    .clientId("<YOUR_CLIENT_ID>")
+	    .clientSecret("<YOUR_CLIENT_SECRET>")
+	    .tenantId("<YOUR_TENANT_ID>");
+
+ClientSecretcredential secondServicePrincipal = new ClientSecretCredential()
+	    .clientId("<YOUR_CLIENT_ID>")
+	    .clientSecret("<YOUR_CLIENT_SECRET>")
+	    .tenantId("<YOUR_TENANT_ID>");
+
+// when an access token is requested, the chain will try each
+// credential in order, stopping when one provides a token
+
+ChainedTokenCredential credentialChain = new ChainedTokenCredential()
+		.addLast(firstServicePrincipal)
+		.addLast(secondServicePrincipal);
+
+// the chain can be used anywhere a credential is required
+
+// The fully qualified host name for the Event Hubs namespace. This is likely to be similar to:
+// {your-namespace}.servicebus.windows.net
+String host = "<< EVENT HUBS HOST >>"
+String eventHubPath = "<< NAME OF THE EVENT HUB >>";
+EventHubClient client = new EventHubClientBuilder()
+    .credential(host, eventHubPath, credentialChain)
+    .build();
+```
+
+## Troubleshooting
+Credentials raise exceptions when they fail to authenticate. `ClientAuthenticationException` has a `message` attribute which
+describes why authentication failed. When raised by `ChainedTokenCredential`, the message collects error messages from each credential in the chain.
+
+## Next steps
+Currently the following client libraries support authenticating with `TokenCredential` and the Azure Identity library.  You can learn more about their use, and find additional documentation on use of these client libraries along samples with can be found in the links below.
+
+- [azure-eventhubs][eventhubs_client_library]
+- [azure-keyvault-keys][keys_client_library]
+- [azure-keyvault-secrets][secrets_client_library]
```

Comment:
shouldn't this list contains azure-storage-blobs and azure-storage-queues as well?

Reply:
For Java, only storage-blob was released.
azure-storage-blob does support Azure Identity, but they currently lack samples demonstrating Azure Identity use case for auth. Once they are added, I will update the README here.